test/compilable/diag20916.d

```diff
@@ -7,7 +7,7 @@ compilable/diag20916.d(42):        instantiated from here: `jump1!(Foo)`
 compilable/diag20916.d(32): Deprecation: function `diag20916.FooBar.toString` is deprecated
 compilable/diag20916.d(37):        instantiated from here: `jump2!(Foo)`
 compilable/diag20916.d(42):        instantiated from here: `jump1!(Foo)`
-compilable/diag20916.d(32): Deprecation: function `diag20916.Bar.toString!().toString` is deprecated
+compilable/diag20916.d(32): Deprecation: template `diag20916.Bar.toString()()` is deprecated
```

Comment:
This function is declared as:
```
 deprecated string toString()() const { return "42"; }
```
If you were to change it to...
```
 deprecated { string toString()() const { return "42"; } }
```
You'd instead get a deprecation on the template.

So this new diagnostic is correct, as both `deprecated template` and `deprecated { template }` now give the same deprecation warning.